src/applications/vaos/utils/appointment.jsx

```diff
@@ -198,9 +214,14 @@ export function getAppointmentDateTime(appt) {
 
   return (
     <>
-      {parsedDate.format('MMMM D, YYYY')} at {parsedDate.format('h:mm')}
-      <span aria-hidden="true"> {parsedDate.format('a')}</span>
-      <span className="sr-only">{parsedDate.format('a')}</span>
+      {parsedDate.format('dddd, MMMM D, YYYY')} at {parsedDate.format('h:mm')}
+      <span aria-hidden="true">
+        {' '}
+        {parsedDate.format('a')} {getAppointmentTimezone(appt)}
```

Comment:
It's not super clear to me, but I believe we have to parse the date we get using the timezone. I believe the times we get back for confirmed appointments are in UTC. And for requests, the data does include a timezone, but I don't know if the time is adjusted for it. We should be able to figure this out by comparing data between the legacy app and ours (especially with the Cheyenne system, since it's not EST).